src/components/elements/List/style.scss

```diff
@@ -0,0 +1,80 @@
+:local {
+  @import '../../../styles/config';
+
+  .List {
+    margin-top: 0;
+    margin-bottom: 0;
+    padding-left: 0;
+    list-style: none;
+
+    > li {
```

Comment:
@suwanny I have thought about your feedback and back to my original approach to use BEM. I think we can achieve the BEM with react `{children}`, but we need to add same `type` property to `<ListItem>` component like as:
```jsx
<List type='instruction'>
  <ListItem type='instruction'>Item 1</ListItem>
  <ListItem type='instruction'>Item 2</ListItem>
  <ListItem type='instruction'>Item 3</ListItem>
</List>
```

So, more change to make mistake from the user if user has typo in type property or different type.